pytest/tests/loadtest/locust/common/base.py

```diff
@@ -410,6 +433,34 @@ def evaluate_rpc_result(rpc_result):
     return result
 
 
+def as_action_error(failure: dict) -> typing.Optional[dict]:
+    if "ActionError" in failure:
```

Comment:
You can rewrite this function by using the default value on `dict.get(key, default)`: `failure.get("ActionError", None)` [1]

[1] https://python-reference.readthedocs.io/en/latest/docs/dict/get.html